app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/module/_toolbar.less

```diff
@@ -141,6 +141,14 @@
 //  Desktop
 //  _____________________________________________
 
+.media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__xs) {
+    .sorter {
+        .sorter-label {
+            display: none;
```

Comment:
Review if hiding the label is the best approach.
Maybe repositioning the label could also be a good solution.
Please, check what works better.

Reply:
@rodrigowebjump 
I hide the label because i saw it in many themes. Even in Magento we hide pager and Grid/List icon in small screen.
So i hide it because i think there is no need to show label on small screen size.

Other solution of this problem can be decreasing font size of label and reducing margin of the sorting dropdown.
![Selection_029](https://user-images.githubusercontent.com/15954089/67653978-e596d980-f971-11e9-933b-583891199d40.png)
![Selection_030](https://user-images.githubusercontent.com/15954089/67653981-e92a6080-f971-11e9-8bf7-52711569e841.png)

Let me know if looks good.

Reply:
I have reduced **.sorter-options** margin from **0 5px 0 7px** to **0 2px 0 2px**